### PR TITLE
Make adjustments to release PR templates

### LINKED ISSUE #219: 🚧Make adjustments to release PR templates
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Make various changes to the **release PR templates** to accommodate the addition of **release issues** and to improve the release process.

### **Refer to image below for more info and context:**
image

---

### Acceptance Criteria

**This issue is finished when:**
- [x] Change the top section **✅Development Checklist✅** to **✅Code Review Checklist✅**
- [x] Remove the bottom section title **✅Code Review Checklist✅**
- ~Make the following changes to the **🌲Branching🌲** section.~
  - [x] ~Create a centered heading named Branching to show off the section~
  - [x] Create 3 check boxes for each of the following:
    - Does the name of the head(source) branch for this pull request have the correct preview branch syntax?
      - [x] Add a default collapsed region with the following about the syntax
        - Syntax: preview/v\<major-number\>.\<minor-number\>.\<patch-number\>-preview.\<prev-number\>
      Example: preview/v1.2.3-preview.4
    - Does the name of the base(target) branch for this pull request have the correct release branch syntax?
      - [x] Add a default collapsed region with the following about the syntax 
        - Syntax: release/v\<major-number\>.\<minor-number\>.\<patch-number\>
    Example: release/v1.2.3
    - Is the head(source) branch for this pull request created from a branch with the correct release branch syntax?
      - [x] Add a default collapsed region with the following about the syntax 
        - Syntax: release/v\<major-number\>.\<minor-number\>.\<patch-number\>
Example: release/v1.2.3
- Remove the sections asking what kinds of changes the release contains and the adding of labels.  Refer to list below on what to remove.
  - [x] 🐛Bugs🐛 Removed
  - [x] 💣Breaking Changes💣 Removed
  - [x] ✨Enhancements✨ Removed
  - [x] ⚙️Workflow (CI/CD> Changes⚙️ Removed
  - [x] 📃Documentation Updates📃 Removed
- [x] Refactor the **🧪Manual Testing🧪** text to the value below:
  - A QA issue has been created and manual QA testing has been performed for the changes being released
- [x]  Remove the checkbox below.  This is now done in the associated **Release ToDo Issue** that is linked to the PR
image
- [x] Add an item somewhere in the **✅Code Review Checklist✅** to add the appropriate release label to the PR.
  - [x] ~Add a table inside of a default collapsed section that describes the type of release labels to add~
- [x] Remove the following check boxes and associated info below them from all of the PR templates except the `feature-pr.md` feature PR template.
image
- [x] Remove the following check box and associated info below it from all of the PR templates.
image

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [x] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

- KinsonDigital/issues-repo#9

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>

<h2 style="font-weight:bold" align="center">✅Development Checklist✅</h2>

<details open><summary>🐛Bugs🐛</summary>

Contains Bug Fix(es)
- [ ] Yes
  - [ ] A bug-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>🧨Breaking Change(s)🧨</summary>

Contains [breaking change(s)](https://docs.microsoft.com/en-us/dotnet/core/compatibility/#modifications-to-the-public-contract)
- [ ] Yes
  - [ ] A breaking-change-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>✨Enhancements✨</summary>

Contains enhancement(s) that add a feature or behavior.
- [ ] Yes
  - [ ] An enhancement-label label has been added to the pull request.
- [x]  No
</details>

<details open><summary>⚙️Workflow (CI/CD) Changes⚙️</summary>

Contains changes to workflow files. These changes can only be done by the project maintainers.
- [ ] Yes
  - [ ] A workflow-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>📃Documentation Updates📃</summary>

Contains changes that require updates to existing **_code_** and/or **_Velaptor product_** documentation.
- [x] Yes
  - [x] I have updated the documentation accordingly.
    - Choose the type of documentation.
    - [ ] Code Documentation
      - [ ] A documentation-code-label label has been added to the pull request.
    - [x] Product Documentation
      - [x] A documentation-product-label label has been added to the pull request.
- [ ] No
</details>

<details open><summary>🧪Unit Testing🧪</summary>

My change requires unit tests to be written.
- [ ] Yes
- [x] No
</details>

<details open><summary>🧪Manual Testing🧪</summary>

I have manually tested my changes using the project named **_VelaptorTesting_**.
- [ ] Yes
- [x] No
</details>

<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] Pull request title matches the title of the linked issue.
- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.
- [x] My code follows the coding style of this project.
  - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
- [x] A preview-label label has been added to the pull request.
- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/preview-release-pr.md [hunk 1]
```diff
@@ -4,112 +4,46 @@
...
-- [ ] Yes
-- [ ] No
-</details>
+      ``` xml
+      Syntax: release/v<major>.<minor>.<patch>
+      Example: release/v1.2.3
+      ```
+    </details>
 
+- [ ] The **_head(source)_** branch for this pull request is created from a **_release_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-release-pr.md:29]: Add colon, remove period
```suggestion
- [ ] The **_head(source)_** branch for this pull request is created from a **_release_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/preview-release-pr.md [hunk 2]
```diff
@@ -4,112 +4,46 @@
...
+  <details closed><summary>Example</summary>
+    
+    ``` xml
+    Syntax: 🚀Release To Preview - v<major>.<minor>.<patch>-preview.<prev-num>
+    Example: 🚀Release To Preview - v1.2.3-preview.4
     ```
-- [ ] I have updated the release notes by creating a new preview release notes file and adding it to the **_./Documentation/ReleaseNotes/PreviewReleases_** folder.
+  </details>
+
+- [ ] A https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Preview%20Release label has been added to the this PR.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-release-pr.md:47]: Remove "the"
```suggestion
- [ ] A https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Preview%20Release label has been added to this PR.
```

File: .github/PULL_REQUEST_TEMPLATE/production-release-pr.md [hunk 3]
```diff
@@ -4,109 +4,32 @@
 -->
 <!--suppress HtmlDeprecatedAttribute -->
 <h1 style="font-weight:bold" align="center">Production Release Pull Request</h1>
+<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
 
+- [ ] The **_head(source)_** branch for this pull request is a **_release_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/production-release-pr.md:9]: Remove period, use colon

```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_release_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/production-release-pr.md [hunk 4]
```diff
@@ -4,109 +4,32 @@
...
-- [ ] No
-</details>
+      ``` xml
+      Syntax: release/v<major>.<minor>.<patch>
+      Example: release/v1.2.3
+      ```
+    </details>
 
+- [ ] The **_base(target)_** branch for this pull request is a **_production_** branch with the name of **_master_**.
+  - A **_base(target)_** branch is the branch that is a source branch being merged into it.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/production-release-pr.md:20]: Sentence structure
```suggestion
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.
```

File: .github/PULL_REQUEST_TEMPLATE/production-release-pr.md [hunk 5]
```diff
@@ -4,109 +4,32 @@
...
 
-<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
+- [ ] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.
 
-<!-- Go over all of the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] Pull request title matches the example below with the correct version.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/production-release-pr.md:24]: Grammar
```suggestion
- [ ] Pull request title matches the example below, using the correct version.
```

File: .github/PULL_REQUEST_TEMPLATE/production-release-pr.md [hunk 6]
```diff
@@ -4,109 +4,32 @@
...
+  <details closed><summary>Example</summary>
+    
+    ``` xml
+    Syntax: 🚀Release To Production - v<major>.<minor>.<patch>
+    Example: 🚀Release To Production - v1.2.3
     ```
-- [ ] I have updated the release notes by creating a new production release notes file and adding it to the **_./Documentation/ReleaseNotes/ProductionReleases_** folder.
+  </details>
+
+- [ ] A https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Production%20Release label has been added to the this PR.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/production-release-pr.md:33]: Remove "the"
```suggestion
- [ ] A https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Production%20Release label has been added to this PR.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — COMMENTED** [review]
All changes confirmed.

**Review by kselena — APPROVED** [review]
